packages/bpk-component-skip-link/examples.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import React from 'react';
+
+import BpkSkipLink from './index';
+
+const DefaultExample = () => (
+  <BpkSkipLink
+    style={{ position: 'absolute' }}
+    href="#main"
+    label="Skip to main content"
+  />
+);
+
+const DocsSiteExample = () => (
+  <BpkSkipLink
+    style={{ position: 'absolute' }}
+    href="#implementation"
+    label="Skip to implementation"
```

Comment:
My bad. It's called `readme`
```suggestion
    href="#readme"
    label="Skip to readme"
```

Reply:
Not according to github 🤔 https://backpack.github.io/components/skip-link/?platform=web#implementation

Reply:
This is also pulled from the live inspection of the code currently deployed

![Screenshot 2021-03-17 at 13 27 24](https://user-images.githubusercontent.com/8831547/111475009-8d0ddb00-8724-11eb-95b4-dc13d57f69c5.png)